apps/communications/contacts/js/contacts.js

```diff
@@ -242,26 +238,24 @@ var Contacts = (function() {
   };
 
   var contactListClickHandler = function originalHandler(id) {
-    initDetails(function onDetailsReady() {
-      ContactsService.get(id, function findCb(contact, fbContact) {
 
-        currentContact = contact;
-        currentFbContact = fbContact;
+    if (!ActivityHandler.currentlyHandling) {
+      window.location.href =
+        '/contacts/views/details/details.html?contact=' + id;
```

Comment:
We should create a method like:
`getContatDetailUrl(id)`

To use it later on, we could potentially change the base url depending on settings, device or anything.